src/views/List.jsx

```diff
@@ -7,11 +7,9 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((item) => (
+					<ListItem name={item.name} key={item.id} />
```

Comment:
Very nice but you actually need see the ListItem component itself in case you need to make some changes there.

Reply:
Hi @amanolar94 Can you please explain what you mean? Thank you.